services/web/server/src/simcore_service_webserver/login/_registration_api.py

```diff
@@ -95,3 +100,19 @@ async def send_account_request_email_to_support(
             template_name,
             f"{support_email=}",
         )
+
+
+async def generate_captcha() -> tuple[str, bytes]:
+    captcha_text = "".join(random.choices(string.digits, k=6))
+    image = ImageCaptcha(width=140, height=45)
+
+    # Generate image
+    data: Image = image.create_captcha_image(
```

Comment:
you do not need to dealloacte all these from memory? .e.g. if you save the image in the BytesIO, shouldn't you deallocate then image then?

Reply:
discussed